GOVERNANCE.md

```diff
@@ -35,23 +35,16 @@ The Operator Framework community will work on developing best practices, fosteri
 ## Governance
 
 ### Operations
-The Operator Framework community is run and managed by the following chairs:
-* Diane Mueller
-* Evan Cordell
-* Joe Lanford
-* Shawn Hurley
-* Daniel Messer
-* Rob Szumski
-* Gerred Dillon
+The Operator Framework community is run and managed by the [community chairs][OWNERS_ALIASES].
 
-Note: The referenced names and chair positions will be edited in-place as chairs are added, removed, or replaced. See the roles of chairs section for more information.
+Note: The referenced names and chair positions will be edited in-place as chairs are added, removed, or replaced.
 
 A dedicated git repository will be the authoritative archive for membership list, code, documentation, and decisions made. The repository, along with this charter, will be hosted at github.com/operator-framework/community.
 
 The mailing list at groups.google.com/forum/#!forum/operator-framework will be used as a place to call for and publish group decisions, and to hold discussions in general.
 
 ### Operator Framework Community Membership
-All active members of the Operator Framework community are listed in the `OWNERS.md` file with their name.
+All active members of the Operator Framework community are listed in the [`OWNERS`][OWNERS_ALIASES].
```

Comment:
This makes the active members only as folks in "community-chairs" and "community-maintainers". Although this PR doesn't make this worse, do we want to also address the definition/reference of the "active owner name" along with editing this sentence?

Reply:
I don't think we can, because the change I am proposing here doesn't change the substance of the document. To change/clarify definitions, I think we need a governance vote.

Reply:
@asmacdo so we're making the set of membership limited to the OWNERS file in COMMUNITY repo. And we are going to update it with the folks from SDK and OLM repos?

Reply:
@asmacdo Ah I see it now. The change above doesn't actually change the membership. It simply changes that instead of being in an `OWNERS.md` file it is in the `OWNERS` file.

Reply:
My original concern is about the potential misperception to how active members are being considered.
After talking with @asmacdo, this PR is just a part of the larger changes and not intended to be referred to a finalized version, then I have less concern with the change.